scripts/components/api-changes-validator/api_usage_generator.test.ts

```diff
@@ -353,7 +353,10 @@ void describe('Api usage generator', () => {
         'samplePackageName',
         apiReportAST
       ).generate();
-      assert.strictEqual(apiUsage.trim(), testCase.expectedApiUsage.trim());
+      assert.strictEqual(
+        apiUsage.replace(/[\r]/g, '').trim(),
```

Comment:
mind adding a comment why are we doing this? 
(presumably due to different EOL on windows).